### PR TITLE
Add m2w64 compilers for windows

### PR DESCRIPTION
**Pull request by twiecki**
It seems like currently we are not installing the right compilers for windows.

### CODE REVIEW

**Comment by twiecki** [comment]
@conda-forge-admin please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,15 +25,18 @@ requirements:
...
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
     - python
     - setuptools
     - six >=1.9.0
     - numpy >=1.9.1
     - scipy >=0.14
-    - pygpu >=0.7.0,<0.8
     - filelock
+    - jax  # [py>=37 and not win]
```
Comment (brandonwillard) [inline recipe/meta.yaml:38]: This isn't a real dependency of our library.

**Review by brandonwillard — CHANGES_REQUESTED** [review]
Thanks for looking into this!  I don't know what the situation is with these Windows dependencies, so I can't really say what's correct or not with those; however, it seems like the PyPy builds are failing, and that makes sense given our non-trivial dependencies and direct use of CPython-based extensions.

**Review by brandonwillard — APPROVED** [review]

**Pull request merged**